slave/process-kypo-portal/bin/process-kypo_portal.sh

```diff
@@ -31,11 +31,16 @@ function process {
 	diff_mv "${FROM_PERUN_GROUPS}" "${DST_FILE_GROUPS}"
 	group_diff=$?
 	
-	if [ $user_diff -eq 0 ] || [ $group_diff -eq 0 ]; then
+	if [ $user_diff -ne 0 ] && [ $group_diff -ne 0 ]; then
+		log_msg I_NOT_CHANGED
```

Comment:
you should do exit 0 here. Otherwise the flow is flow is a bit more complex (with the 'else') and the files in /tmp get always deleted.